admin_guide/limits.adoc

```diff
@@ -181,6 +210,98 @@ Across all containers in a pod, the following must hold true:
 |===
 // end::admin_limits_pod_limits[]
 
+[[image-limits]]
+=== Image Limits
+
+// tag::admin_limits_image_limits[]
+
+*Supported Resources:*
+
+* Storage
+
+*Resource type name:*
+
+- `openshift.io/Image`
+
+[cols="3a,8a",options="header"]
+|===
+|Constraint |Behavior
+
+|`*Max*`
+|`image.dockerImageMetadata.size` less than or equal to Max[resource]
+|===
+
+[NOTE]
+====
+To prevent blobs exceeding the limit from being uploaded to the registry, the
+registry must be configured to enforce quota. An environment varibale
+`*REGISTRY_MIDDLEWARE_REPOSITORY_OPENSHIFT_ENFORCEQUOTA*` must be set to
+`*true*` which is done by default for new deployments. To update older
+deployment config, refer to
+link:../install_config/upgrading/manual_upgrades.html#enforcing-quota-in-the-registry[Enforcing
+quota in the Registry].
+
+The image size isn't always available in the manifest of uploaded image. This
+is especially the case for images built with docker of version 1.10 or higher
```

Comment:
The fix may fall behind. I'll happily remove the comment once fixed. Until then I'd rather keep it.